tests/robot-tests/scripts/run_tests_pipeline.py

```diff
@@ -23,6 +23,7 @@ def run_tests_pipeline():
         raise Exception(f"Invalid environment provided: {args.env}. Valid environments: {valid_environments}")
 
     subprocess.check_call(["google-chrome-stable", "--version"])
+    subprocess.check_call(["python", "--version"])
```

Comment:
No need for this check afaik. We install a hardcoded python version via a pipeline task so not sure why this is needed

Reply:
I added it as more of a convenience, to save having to click around. I'm going to leave it, but if others feel strongly about it, I'll remove it.